isototest/src/action.rs

```diff
@@ -20,131 +39,273 @@ use vnc::{client::VncClient, ClientKeyEvent, VncError, X11Event};
+        '9' => Ok(KeyCode::Key9),
+        '0' => Ok(KeyCode::Key0),
+        'A' => Ok(KeyCode::A),
+        'B' => Ok(KeyCode::B),
+        'C' => Ok(KeyCode::C),
+        'D' => Ok(KeyCode::D),
+        'E' => Ok(KeyCode::E),
+        'F' => Ok(KeyCode::F),
+        'G' => Ok(KeyCode::G),
+        'H' => Ok(KeyCode::H),
+        'I' => Ok(KeyCode::I),
+        'J' => Ok(KeyCode::J),
+        'K' => Ok(KeyCode::K),
+        'L' => Ok(KeyCode::L),
+        'M' => Ok(KeyCode::M),
+        'N' => Ok(KeyCode::N),
+        'O' => Ok(KeyCode::O),
+        'P' => Ok(KeyCode::P),
+        'Q' => Ok(KeyCode::Q),
+        'R' => Ok(KeyCode::R),
+        'S' => Ok(KeyCode::S),
+        'T' => Ok(KeyCode::T),
+        'U' => Ok(KeyCode::U),
+        'V' => Ok(KeyCode::V),
+        'W' => Ok(KeyCode::W),
+        'X' => Ok(KeyCode::X),
+        'Y' => Ok(KeyCode::Y),
+        'Z' => Ok(KeyCode::Z),
+        'a' => Ok(KeyCode::a),
+        'b' => Ok(KeyCode::b),
+        'c' => Ok(KeyCode::c),
+        'd' => Ok(KeyCode::d),
+        'e' => Ok(KeyCode::e),
+        'f' => Ok(KeyCode::f),
+        'g' => Ok(KeyCode::g),
+        'h' => Ok(KeyCode::h),
+        'i' => Ok(KeyCode::i),
+        'j' => Ok(KeyCode::j),
+        'k' => Ok(KeyCode::k),
+        'l' => Ok(KeyCode::l),
+        'm' => Ok(KeyCode::m),
+        'n' => Ok(KeyCode::n),
+        'o' => Ok(KeyCode::o),
+        'p' => Ok(KeyCode::p),
+        'q' => Ok(KeyCode::q),
+        'r' => Ok(KeyCode::r),
+        's' => Ok(KeyCode::s),
+        't' => Ok(KeyCode::t),
+        'u' => Ok(KeyCode::u),
+        'v' => Ok(KeyCode::v),
+        'w' => Ok(KeyCode::w),
+        'x' => Ok(KeyCode::x),
+        'y' => Ok(KeyCode::y),
+        'z' => Ok(KeyCode::z),
+        ' ' => Ok(KeyCode::SPACE),
+        '!' => Ok(KeyCode::ExcMrk),
+        '@' => Ok(KeyCode::At),
+        '#' => Ok(KeyCode::Pound),
+        '$' => Ok(KeyCode::Dollar),
+        '%' => Ok(KeyCode::Percent),
+        '^' => Ok(KeyCode::Caret),
+        '&' => Ok(KeyCode::And),
+        '*' => Ok(KeyCode::Ast),
+        '(' => Ok(KeyCode::LRBrace),
+        ')' => Ok(KeyCode::RRBrace),
+        '-' => Ok(KeyCode::Minus),
+        '_' => Ok(KeyCode::UScore),
+        '=' => Ok(KeyCode::Equals),
+        '+' => Ok(KeyCode::Plus),
+        '[' => Ok(KeyCode::LBracket),
+        ']' => Ok(KeyCode::RBracket),
+        '{' => Ok(KeyCode::LCrlBrace),
+        '}' => Ok(KeyCode::RCrlBrace),
+        '\\' => Ok(KeyCode::BckSlash),
+        '|' => Ok(KeyCode::Pipe),
+        ';' => Ok(KeyCode::SColon),
+        ':' => Ok(KeyCode::Colon),
+        '\'' => Ok(KeyCode::Apo),
+        '"' => Ok(KeyCode::DblQuote),
+        ',' => Ok(KeyCode::Comma),
+        '.' => Ok(KeyCode::Period),
+        '/' => Ok(KeyCode::FwdSlash),
+        '<' => Ok(KeyCode::LThan),
+        '>' => Ok(KeyCode::GThan),
+        '?' => Ok(KeyCode::Question),
+        '\n' => Ok(KeyCode::LineFeed),
+        '`' => Ok(KeyCode::GraveAcc),
+        '~' => Ok(KeyCode::Tilde),
```

Comment:
Ah, you saw it too xD, I saw that it was missing

Reply:
Yes, according to my tests, it works fine now :)